mkcloudruns/hacks/conduitresolver.rb

```diff
@@ -0,0 +1,342 @@
+#
```

Comment:
this should rather be added to the crowbar / crowbar-core repository and installed + made available for any customer. no need to keep that in a hacks/ subdirectory.